app/javascript/packs/actions/initialState.js

```diff
@@ -29,6 +29,9 @@ const mapFilterKeys = (objs, keys) => {
 };
 
 const formatState = {
+  user: function(user) {
+      return filterKeys(user[0], ['id', 'email', 'first_name', 'last_name']);
```

Comment:
what is `user[0]`? this seems to be breaking abstraction barriers

Reply:
what would you recommend I do instead? I just copy pasted that since user was nested. I could have a function called format which would just return object[0] but that seems redundant